easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -196,6 +196,7 @@ def __init__(self, path, extra_options=None, build_specs=None, validate=True, hi
             self.validate(check_osdeps=build_option('check_osdeps'))
 
         # filter hidden dependencies from list of dependencies
+        self.orig_dependencies = None
```

Comment:
mainly to silence pylint, it's mostly a style issue

I want to make the actual copy in `filter_hidden_deps`, where `dependencies` is being modified (potentially)